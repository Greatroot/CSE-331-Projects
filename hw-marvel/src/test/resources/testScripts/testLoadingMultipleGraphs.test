# Tests loading multiple graphs from the same tsv and modifying those graphs.

LoadGraph g1 testData.tsv
LoadGraph g2 testData.tsv

AddNode g1 A
AddNode g1 B

AddNode g2 A
AddNode g2 C
AddNode g2 D

ListNodes g1
ListNodes g2