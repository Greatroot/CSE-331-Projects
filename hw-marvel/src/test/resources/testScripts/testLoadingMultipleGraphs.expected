# Tests loading multiple graphs from the same tsv and modifying those graphs.

loaded graph g1
loaded graph g2

added node A to g1
added node B to g1

added node A to g2
added node C to g2
added node D to g2

g1 contains: A B n1 n10 n11 n12 n13 n2 n3 n4 n5 n6 n7 n8 n9
g2 contains: A C D n1 n10 n11 n12 n13 n2 n3 n4 n5 n6 n7 n8 n9